src/main/java/seedu/address/model/applicant/Id.java

```diff
@@ -0,0 +1,50 @@
+package seedu.address.model.applicant;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents an Applicant's Id in the address book.
+ */
+public class Id {
+
+    public static final String MESSAGE_CONSTRAINTS = "Id can take any values, and it should not be blank";
```

Comment:
Maybe change the MESSAGE_CONSTRAINT to "Id should only be a positive integer, and it should not be blank".